UnityProject/Assets/Scripts/Medical/CloningConsole.cs

```diff
@@ -92,6 +100,12 @@ public void Scan()
 			CreateRecord(mob, playerScript);
 			scanner.statusString = "Subject successfully scanned.";
 		}
+		else {
+			if(!scanner) scanner.statusString = "Scanner not found.";
+			else if(!scanner.RelatedAPC) scanner.statusString = "Scanner not connected to APC.";
```

Comment:
you can also make this a switch statement
https://www.csharp-examples.net/switch/

Reply:
I don't think a switch statement would work here since I'm checking through a series of booleans (converted via the '!' operator) rather than iterating over integral types. It's possible to force using a switch statement by converting the booleans into integers and then bitmasking them, but that would be even harder to read.

Reply:
ok, forgot it has to be static/permanent thing